source/frontend/src/features/mapSideBar/property/tabs/propertyDetails/detail/propertyOperation/SubdivisionContainer.tsx

```diff
@@ -0,0 +1,103 @@
+import { Dictionary, groupBy } from 'lodash';
+import { useCallback, useEffect, useState } from 'react';
+
+import { usePimsPropertyRepository } from '@/hooks/repositories/usePimsPropertyRepository';
+import { usePropertyOperationRepository } from '@/hooks/repositories/usePropertyOperationRepository';
+import { ApiGen_Concepts_Property } from '@/models/api/generated/ApiGen_Concepts_Property';
+import { UtcIsoDateTime } from '@/models/api/UtcIsoDateTime';
+import { exists, unique } from '@/utils';
+
+import { SubdivisionView } from './SubdivisionView';
+
+interface ISubdivisionContainer {
+  propertyId: number;
+}
+
+interface OperationSet {
+  operationDateTime: UtcIsoDateTime;
+  sourceProperties: ApiGen_Concepts_Property[];
+  destinationProperties: ApiGen_Concepts_Property[];
+}
+
+export const SubsivisionContainer: React.FunctionComponent<ISubdivisionContainer> = ({
+  propertyId,
+}) => {
+  const View = SubdivisionView;
+
+  const [operationDictionary, setOperationDictionary] = useState<Dictionary<OperationSet>>({});
+
+  const { getPropertyWrapper } = usePimsPropertyRepository();
+  const getPropertyExecute = getPropertyWrapper.execute;
+
+  const { getPropertyOperations } = usePropertyOperationRepository();
+  const getPropertyOperationsExecute = getPropertyOperations.execute;
+
+  const fetchOperations = useCallback(
+    async (propertyId: number) => {
+      const results = await getPropertyOperationsExecute(propertyId);
+
+      if (exists(results)) {
+        const resultGroup = groupBy(results, x => x.propertyOperationNo);
+
+        for (const key in resultGroup) {
+          const result = resultGroup[key];
+
+          const sourceIds = result
+            .map(po => po.sourcePropertyId)
+            .filter(exists)
+            .filter(unique);
+          const destinationIds = result
+            .map(po => po.destinationPropertyId)
+            .filter(exists)
+            .filter(unique);
+
+          const sourcePromises: Promise<ApiGen_Concepts_Property | undefined>[] = [];
+          const destinationPromises: Promise<ApiGen_Concepts_Property | undefined>[] = [];
+
+          for (let i = 0; i < sourceIds.length; i++) {
+            sourcePromises.push(getPropertyExecute(sourceIds[i]));
+          }
+          for (let i = 0; i < destinationIds.length; i++) {
+            destinationPromises.push(getPropertyExecute(destinationIds[i]));
+          }
+
+          const retrievedSources = await Promise.all(sourcePromises);
+          const retrievedDestinations = await Promise.all(destinationPromises);
```

Comment:
**nit:** something to thing about/ tech-debt: this will (potentially) make a lot of requests to the GetProperty api endpoint. We should consider a "batch" get endpoint where we pass an array of Ids and get back the matching properties